source/physics/include/GateGammaSourceModel.hh

```diff
@@ -0,0 +1,35 @@
+#ifndef GATEGAMMASOURCEMDEL_HH
```

Comment:
Can you add the header similar what we have in framework e.g (https://github.com/JPETTomography/j-pet-framework/blob/master/JPetPM/JPetPM.h) to all your new files, but changing the licence to GPLG since it s not compatible with Apache2 and original gate uses LPLG.